buffer/src/main/java/io/netty/buffer/PooledByteBufAllocator.java

```diff
@@ -187,6 +186,12 @@ public void run() {
     public static final PooledByteBufAllocator DEFAULT =
             new PooledByteBufAllocator(PlatformDependent.directBufferPreferred());
 
+    // Default allocator without using thread-local.
+    public static final PooledByteBufAllocator DEFAULT_NO_TL =
```

Comment:
I need to look more into the idea of the PR in general but I don't think we should add just another static instance. Adding the `DEFAULT` in the first place was a failure and we should better not have done it because it can cause memory bloat etc.

Reply:
It's fine to remove it, but I sincerely wondered why the `DEFAULT` caused the memory bloat, it seems a static one which supposed to be initialized only once when use it?